lib/nerves_hub_link/health_check/default_report.ex

```diff
@@ -0,0 +1,122 @@
+defmodule NervesHubLink.HealthCheck.DefaultReport do
+    alias NervesHubLink.HealthCheck.Report
+    use NervesHubLink.HealthCheck.Report
+
+    @impl Report
+    def device_id, do: Nerves.Runtime.serial_number()
+    
+    @impl Report
+    def timestamp, do:
+        DateTime.utc_now()
+        |> DateTime.to_iso8601()
+
+    @impl Report
+    def metadata do
+        # TODO: What else?
+        %{
+            "arch" => KV.get_active("nerves_fw_architecture"),
+            "platform" => KV.get_active("nerves_fw_platform"),
+            "product" => KV.get_active("nerves_fw_product"),
+            "version" => KV.get_active("nerves_fw_version"),
+            "uuid" => KV.get_active("nerves_fw_uuid"),
+        }
+        |> Map.merge(metadata_from_config())
+    end
```

Comment:
I'd skip the `device_id`, `timestamp` and the default `metadata` information too.

All this data is known by Hub when the connection is established.